examples/src/main/java/com/mapbox/navigation/examples/activity/OffboardRouterActivityKt.kt

```diff
@@ -168,6 +169,7 @@ class OffboardRouterActivityKt : AppCompatActivity(),
             }.build()
 
             offboardRouter?.getRoute(options, this@OffboardRouterActivityKt)
+            Handler().postDelayed({ offboardRouter?.cancel() }, 50)
```

Comment:
Do we want to leave this to showcase that `cancel` is working 👌?

Reply:
whoops, that was a testing setup